deepspeed/runtime/fp16/loss_scaler.py

```diff
@@ -190,6 +191,8 @@ def update_scale(self, overflow):
             self.last_overflow_iter = self.cur_iter
         else:
             if self.consecutive_hysteresis:
+                hysteresis_msg = f"Consecutive hysteresis is enabled. Restoring hysteresis to {self.delayed_shift}"
+                logger.info(hysteresis_msg)
```

Comment:
Can you wrap the logger.info around an `if dist.get_rank() == 0:` so we only see one log instead of `world_size`?

Reply:
Resolved in https://github.com/microsoft/DeepSpeed/pull/3553/commits/d3a48460422ba916b1ae21f3c392274ff3d6991d